ui/button.js

```diff
@@ -24,31 +24,14 @@
 	}
 }(function( $ ) {
 
-var lastActive,
-	baseClasses = "ui-button ui-widget ui-state-default ui-corner-all",
-	typeClasses = "ui-button-icons-only ui-button-icon-only ui-button-text-icons ui-button-text-icon-primary ui-button-text-icon-secondary ui-button-text-only",
-	formResetHandler = function() {
+var formResetHandler = function() {
 		var form = $( this );
 		setTimeout(function() {
```

Comment:
I know this has been here for several years, and I figured this out in about a minute, but it wasn't immediately obvious to me why we had this timeout. Let's add a comment that says "Wait for the form refresh to actually occur before refreshing the buttons"